studio/style.css

```diff
@@ -47,22 +47,21 @@ sp-underlay + sp-dialog {
 
 #editor {
     position: fixed;
-    bottom: 1em;
-    left: var(--editor--left);
-    right: var(--editor--right);
-    top: var(--editor--top);
-    bottom: var(--editor--bottom);
-    height: 800px;
-}
-
-#editor sp-dialog {
-    width: 600px;
-    overflow-y: scroll;
+    bottom: 0;
+    top: 0;
+    left: var(--editor-left);
+    right: var(--editor-right);
+    height: 100vh;
+    width: 440px;
+    background-color: var(--spectrum-white);
```

Comment:
weird white spaces?

Reply:
huh? I'm not sure what's going on here. The lines are all tabbed out, and my git extension isn't registering any changes when I try to "remove" them.

Reply:
I see this, not sure what it is:
![image](https://github.com/user-attachments/assets/708f5130-5634-4351-8ecb-d18f5cc11ec5)

Reply:
I see it too here on git, but not sure why that is. 

https://github.com/user-attachments/assets/8e924cfa-15b7-4aa0-9851-70743b16119c